test/browser/compat.test.js

```diff
@@ -0,0 +1,54 @@
+import { createElement, render } from 'preact';
+import { setupScratch, teardown } from '../../test/_util/helpers';
+
+import React from 'preact/compat'; // eslint-disable-line
+
+describe('imported compat', () => {
```

Comment:
Since these tests are for compat, we should move them to the compat test suite :+1:

Reply:
moved file